src/components/SearchBar/SearchBar.tsx

```diff
@@ -1,25 +1,37 @@
 import React, { useEffect, useState } from "react";
 
+import type { Filter } from "../../hooks/useMembers";
+
+import style from "./SearchBar.module.css";
+
 interface SearchBarProps {
+  filter: Filter;
   onSearch: (name: string, cohort: number | null) => void;
```

Comment:
요기 API 디자인이 살짝 의아합니다. `useMembers ` 모듈에서 `Filter` 타입을 불러오셨다면, `filter`와 `onSearch` prop에 일관적으로 사용하셔야 하지 않을까요?

```suggestion
  onSearch: (filter: Filter) => void;
```

Reply:
오, 그러는 게 더 좋겠네요. 놓쳤습니다!

Reply:
이거 반영해서 병합하겠습니다~